src/templates/zh-CN/2020/methodology.html

```diff
@@ -0,0 +1,475 @@
+
+        <p>
+          Web Almanac 是一个由 <a href="https://httparchive.org">HTTP Archive</a> 组织的项目. HTTP Archive 于2010年由Steve Souders创立，其任务是跟踪web是如何构建的。它按月评估数以百万计的网页，并使它的TB级别的元数据可供<a href="https://httparchive.org/faq#how-do-i-use-bigquery-to-write-custom-queries-over-the-data">BigQuery</a>分析。
+        </p>
+
+        <p>
+          Web Almanac网络年鉴的使命是成为一个关于网络状态的年度公共知识库。我们的目标是通过让主题专家提供符合实际情况的见解，使HTTP Archive的数据仓库更容易被Web社区使用。
+        </p>
+
+        <p>
+          2020年版 Web Almanac 分为四个部分：内容、体验、发布和分发。在每个部分中，有几个章节从不同的角度探讨其总体主题。例如，第二部分在性能、安全和可访问性等章节中探讨了用户体验的不同角度。
+        </p>
+    </section>
+
+    <section>
+        <h2 id="dataset"><a href="#dataset" class="anchor-link">关于数据集</a></h2>
+
+        <p>
+          HTTP Archive 数据集每月持续更新新的数据。对于2020年版的Web Almanac，除非本章另有说明，否则所有的指标都来自于2020年8月的抓取。这些结果<a href="https://github.com/HTTPArchive/httparchive.org/blob/master/docs/gettingstarted_bigquery.md">可以用<code>2020_08_01</code>为前缀在BigQuery公开的查询</a>。
+        </p>
+
+        <p>
+          Web Almanac中列出的所有指标都可以通过BigQuery上的数据集公开重现。您可以在我们的<a href="https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/sql/2020">GitHub 仓库</a>浏览我们的所有章节使用的查询。
+        </p>
+
+        <p class="note">
+          请注意，其中一些查询相当大，如果你自己运行可能比较<a href="https://cloud.google.com/bigquery/pricing">昂贵</a> ，因为BigQuery是按TB收费。为了帮助控制你的开支，请看 Tim Kadlec的文章 <a href="https://timkadlec.com/remembers/2019-12-10-using-bigquery-without-breaking-the-bank/">在不破产的情况下来使用BigQuery</a>。
+        </p>
+
+        <p>
+          例如，要了解每个桌面和移动页面的JavaScript字节数的中位数，请参阅 <a href="https://github.com/HTTPArchive/almanac.httparchive.org/blob/main/sql/2019/01_JavaScript/01_01b.sql">01_01b.sql</a>:
+        </p>
+
+        <div class="code-block floating-card">
+          <pre><code><span class="code-comment">#standardSQL
+# 01_01b: Distribution of JS bytes by client</span>
+<span class="code-keyword">SELECT</span>
+  percentile,
+  _TABLE_SUFFIX <span class="code-keyword">AS</span> client,
+  <span class="code-function">APPROX_QUANTILES</span>(<span class="code-function">ROUND</span>(bytesJs <span class="code-keyword">/</span> <span class="code-number">1024</span>, <span class="code-number">2</span>), <span class="code-number">1000</span>)[<span class="code-function">OFFSET</span>(percentile <span class="code-keyword">*</span> <span class="code-number">10</span>)] <span class="code-keyword">AS</span> js_kbytes
+<span class="code-keyword">FROM</span>
+  <span class="code-string">`httparchive.summary_pages.2019_07_01_*`</span>,
+  <span class="code-function">UNNEST</span>([<span class="code-number">10</span>, <span class="code-number">25</span>, <span class="code-number">50</span>, <span class="code-number">75</span>, <span class="code-number">90</span>]) <span class="code-keyword">AS</span> percentile
+<span class="code-keyword">GROUP BY</span>
+  percentile,
+  client
+<span class="code-keyword">ORDER BY</span>
+  percentile,
+  client</code></pre>
```

Comment:
Changed in #1594 

```suggestion
          {# To generate this markup temporarily add a ```sql code block to a chapter and generate that chapter and you'll get the HTML #}
          <pre><code class="sql language-sql"><span class="comment">#standardSQL</span>
<span class="comment"># 01_01b: Distribution of JS bytes by client</span>
<span class="keyword">SELECT</span>
  percentile,
  _TABLE_SUFFIX <span class="keyword">AS</span> client,
  <span class="function call">APPROX_QUANTILES</span>(<span class="function call">ROUND</span>(bytesJs / <span class="constant numeric">1024</span>, <span class="constant numeric">2</span>), <span class="constant numeric">1000</span>)[<span class="function call">OFFSET</span>(percentile <span class="keyword operator">*</span> <span class="constant numeric">10</span>)] <span class="keyword">AS</span> js_kbytes
<span class="keyword">FROM</span>
  <span class="string">`httparchive.summary_pages.2019_07_01_*`</span>,
  <span class="function call">UNNEST</span>([<span class="constant numeric">10</span>, <span class="constant numeric">25</span>, <span class="constant numeric">50</span>, <span class="constant numeric">75</span>, <span class="constant numeric">90</span>]) <span class="keyword">AS</span> percentile
<span class="keyword">GROUP</span> <span class="keyword">BY</span>
  percentile,
  client
<span class="keyword">ORDER</span> <span class="keyword">BY</span>
  percentile,
  client</code></pre>
```